tools/rate-limit-test/rate-limit-test.py

```diff
@@ -0,0 +1,42 @@
+import argparse
```

Comment:
Let's add a `README.md` or some docs here explaining what this script does (which endpoints is testing, etc.).